integration_tests/tests/apply/risks_and_needs/risk-of-serious-harm/rosh_summary.cy.ts

```diff
@@ -11,32 +11,52 @@
 //  Scenario: view 'RoSH summary' page
 //    Then I see the "RoSH summary" page
 //
+//  Scenario: view risk summary data
+//    Given there is risk summary data
+//    Then I see the data presented on the page
+//
+//  Scenario: view 'unknown RoSH' card
+//    Given there is no risk summary data
+//    Then I see the unknown RoSH card
+//
 //  Scenario: navigate to next page in "Risk of serious harm" task
-//    When I continue to the next task / page
+//    When I add an additional comment
+//    And I continue to the next task / page
 //    Then I see the "Risk to others" page
 
 import Page from '../../../../pages/page'
-import { personFactory, applicationFactory } from '../../../../../server/testutils/factories/index'
+import { personFactory, applicationFactory, roshRisksFactory } from '../../../../../server/testutils/factories/index'
 import RoshSummaryPage from '../../../../pages/apply/risks-and-needs/risk-of-serious-harm/roshSummaryPage'
 import RiskToOthersPage from '../../../../pages/apply/risks-and-needs/risk-of-serious-harm/riskToOthersPage'
 
 context('Visit "RoSH summary" page', () => {
   const person = personFactory.build({ name: 'Roger Smith' })
+  const dateOfOasysImport = '2023-09-18'
+  const risks = { ...roshRisksFactory.build(), dateOfOasysImport }
 
   beforeEach(function test() {
     cy.task('reset')
     cy.task('stubSignIn')
     cy.task('stubAuthUser')
 
     cy.fixture('applicationData.json').then(applicationData => {
-      applicationData['risk-of-serious-harm'] = {}
+      applicationData['risk-of-serious-harm'].summary = risks
```

Comment:
I was thinking going forward to have the `applicationData.json` file as an example of a full application with as much data as possible - it will be a handy reference for us when we want to check data structures and ensure we're rendering everything in the `check your answers`. So rather than adding the data in the test it would be in the data by default, and then the test can just `delete` keys on the data to test the unhappy path. Does that make sense?

Reply:
yes, good idea, will change